jpa/src/main/java/org/jboss/as/jpa/container/TransactionScopedEntityManager.java

```diff
@@ -22,6 +22,8 @@
 
 package org.jboss.as.jpa.container;
 
+import static org.jboss.as.jpa.messages.JpaLogger.JPA_LOGGER;
```

Comment:
Is there a change that you wanted to include in this file other than changing the JPA_LOGGER import line?